### PR TITLE
Add Dockerfiles for downstream builds (PROJQUAY-2979)

### PR DESCRIPTION
**Pull request by HammerMeetNail**

### CODE REVIEW

File: Dockerfile.online [hunk 1]
```diff
@@ -0,0 +1,70 @@
+ARG EE_BASE_IMAGE=registry.redhat.io/ansible-automation-platform-20-early-access/ee-minimal-rhel8
```
Comment (jonathankingfc) [inline Dockerfile.online:1]: What if we named this Dockerfile.base, and then had the Dockerfile.offline file be built off of this?

File: ansible-runner/context/_build/requirements.txt [hunk 2]
```diff
@@ -1,2 +1 @@
-ansible
-numpy
\ No newline at end of file
+ansible==5.0.1
```
Comment (jonathankingfc) [inline ansible-runner/context/_build/requirements.txt:1]: Do we need this dependency still?

Comment (HammerMeetNail) [reply ansible-runner/context/_build/requirements.txt:1]: Can probably remove the requirements. Let me do that

File: Dockerfile.online (hunk 1 above)
Comment (HammerMeetNail) [reply Dockerfile.online:1]: Will take care of this in a future Jira

**Review by jonathankingfc — APPROVED** [review]
LGTM

**Pull request merged**